### DISCUSSION TITLE
Tuning parameters explained

### DISCUSSION THREAD

**Discussion by AlexaBennett** [adw96/DivNet#137] [General]
Could someone explain the tuning parameters; EMiter, EMburn, MCiter, MCburn?

**Comment by mooreryan** [comment]
Section 4.1 of the [manuscript](https://academic.oup.com/biostatistics/article/23/1/207/5841114#326929368) has the info about those.

Here's how those parameters you mention go with stuff in the paper:

- `EMiter`: the number of expectation maximization rounds
- `EMburn`: how many of the first few rounds to use for "burn-in"
- `MCiter`: the number of monte-carlo iterations for estimating the distribution
- `MCburn`: how many of the first few monte-carlo iterations are for burn-in
